src/frontend/Komponenter/Behandling/Aktivitet/AktivitetArbeid/AktivitetArbeidInfo.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FC } from 'react';
+import { IAktivitet } from '../../../../App/typer/overgangsstønad';
+import { GridTabell } from '../../../../Felles/Visningskomponenter/GridTabell';
+import { SeksjonWrapper } from '../../../../Felles/Visningskomponenter/SeksjonWrapper';
+import { ArbeidstakerLønnsmottakerSomFrilanser } from '../Aktivitet/ArbeidstakerLønnsmottakerSomFrilanser';
+import SelvstendigNæringsdrivendeEllerFrilanser from '../Aktivitet/SelvstendigNæringsdrivendeEllerFrilanser';
+import { Stønadstype } from '../../../../App/typer/behandlingstema';
+import Aksjeselskap from '../Aktivitet/Aksjeselskap';
+import { Søknadsgrunnlag } from '../../../../Felles/Ikoner/DataGrunnlagIkoner';
+import { ArbeidssituasjonTilTekst, EArbeidssituasjon, EErIArbeid } from '../Aktivitet/typer';
+import { Element, Normaltekst } from 'nav-frontend-typografi';
+
+interface Props {
+    aktivitet: IAktivitet;
+    skalViseSøknadsdata: boolean;
+    stønadstype: Stønadstype;
+}
+
+const AktivitetArbeidInfo: FC<Props> = ({ aktivitet, skalViseSøknadsdata, stønadstype }) => {
+    const { arbeidsforhold, selvstendig, aksjeselskap, virksomhet, erIArbeid } = aktivitet;
+
+    return (
+        <>
+            <SeksjonWrapper>
+                {skalViseSøknadsdata && erIArbeid === EErIArbeid.NeiFordiJegErSyk && (
```

Comment:
Er det interessant å vise info for når man valgt `EErIArbeid.JA`?

Reply:
Dette er et enten-eller-valg. Enten er du i arbeid og har gjort flere valg, eller så er du syk og ikke gjør arbeidsvalg.